src/test/scala/scalismo/faces/FacesTestSuite.scala

```diff
@@ -28,11 +28,13 @@ class FacesTestSuite extends FunSpec with Matchers {
 
   def randomRGB = RGB(rnd.scalaRandom.nextDouble(), rnd.scalaRandom.nextDouble(), rnd.scalaRandom.nextDouble())
 
+  def randomsRGB: sRGB = sRGB(rnd.scalaRandom.nextDouble(), rnd.scalaRandom.nextDouble(), rnd.scalaRandom.nextDouble())
```

Comment:
The sRGB makes the parsing of usual CamelCase a bit hard. But i am not sure if SRGB would improve the readability of the code.